.circleci/config.yml

```diff
@@ -51,37 +57,44 @@ workflows:
       - vue v3:
           requires:
             - build
-      - examples:
-          requires:
-            - build
-      - e2e tests:
-          context: fx-libraries
-          requires:
-            - examples
       - release if needed:
```

Comment:
should the release possibly run e2e as well? I can see the pros and cons of both

Reply:
Making it a prerequisite to `release if needed` would work if we filter the job to be run on tags, but since we're now pushing multiple tags at once, it would run unnecessarily multiple times. Alternatively, we could in the end run e2e tests on master pipelines as well as in a daily schedule. It would still be better than having it run for all branches. Wdyt?

Reply:
I think running it on master may be the most "correct"